config/environment.rb

```diff
@@ -3,3 +3,6 @@
 
 # Initialize the Rails application.
 Packet::Application.initialize!
+
+# Haddock Dictionary for Password Generation
+Haddock::Password.diction = "words.txt"
```

Comment:
Why was this change needed? Where is words.txt?

Reply:
This change was needed in order to get Haddock to properly generate passwords. Words.txt is downloaded separately into the repo's root directory.